src/Nfts/Nfts.tsx

```diff
@@ -0,0 +1,262 @@
+import {useNavigation} from '@react-navigation/native'
+import React, {ReactNode} from 'react'
+import {defineMessages, useIntl} from 'react-intl'
+import {Image, RefreshControl, ScrollView, StyleSheet, Text, View} from 'react-native'
+import {SafeAreaView} from 'react-native-safe-area-context'
+
+import noNftsImage from '../assets/img/no-nft.png'
+import {Icon, Spacer, StatusBar} from '../components'
+import {useNfts} from '../hooks'
+import {WalletStackRouteNavigation} from '../navigation'
+import {useSelectedWallet} from '../SelectedWallet'
+import {ImageGallery, SkeletonGallery} from './ImageGallery'
+
+type Props = {
+  search?: string
+}
+
+export const Nfts = ({search}: Props) => {
+  const searchTermLowerCase = (search ?? '').toLowerCase()
+  const wallet = useSelectedWallet()
+  const {nfts, isLoading, refetch, isRefetching, isError} = useNfts(wallet)
+  const navigation = useNavigation<WalletStackRouteNavigation>()
+  const filteredNFTs =
+    searchTermLowerCase.length > 0 && nfts.length > 0
+      ? nfts.filter((n) => n.name.toLowerCase().includes(searchTermLowerCase))
+      : nfts
+
+  const showDetails = (id: string) => navigation.navigate('nft-details-routes', {screen: 'nft-details', params: {id}})
```

Comment:
convention when navigating
```suggestion
  const navigateToDetails = (id: string) => navigation.navigate('nft-details-routes', {screen: 'nft-details', params: {id}})
```